### ISSUE TITLE
Upgrade Catalina to Monterey

### ISSUE THREAD

**Issue by yassermohammad** [foxlet/macOS-Simple-KVM#522]
Hello everyone 
I have upgraded my OS to Monterey but after restart vm, the OS still is Catalina?
How can I update to Monterey OS?

**Comment by PerryWerneck** [comment]
Same problem here.

**Comment by TamFishe** [comment]
> Hello everyone 
> I have upgraded my OS to Monterey but after restart vm, the OS still is Catalina?
> How can I update to Monterey OS?

Well, you cannot upgrade macOS due to the fact that Clover boot manager blocks them up. That's why in hackintoshing a pc is good with opencore...

**Comment by notAperson535** [comment]
@yassermohammad You can use my fork: https://github.com/notAperson535/OneClick-macOS-Simple-KVM to do it. Just transfer your main .qcow2 to the newly downloaded folder and rename it to macOS.qcow2. Then run basic.sh

**Comment by yassermohammad** [comment]
@notAperson535 but it needs to download the last OS version again

**Comment by yassermohammad** [comment]
I did it already and it is about 12GB
I dont want to download it again

**Comment by notAperson535** [comment]
@yassermohammad I just updated the readme to add instructions to use your own BaseSystem.img file. If this is not the answer you were looking for, tell me what file needs to be redownloaded.

**Comment by yassermohammad** [comment]
@notAperson535 it works, thank you so mach

**Comment by notAperson535** [comment]
@yassermohammad you got monterey?

**Comment by yassermohammad** [comment]
@notAperson535 yes, but there are many problems now,
I cant share files between host and guest or USB does not detected 
do you have any idea?

**Comment by notAperson535** [comment]
@yassermohammad No, but I think Monterey just doesn't like to be on different hardware. I just used Big Sur as it works better. All of the main features in Monterey are either video call features or Safari, which you can get in Big Sur. If you need USB I would recommend Big Sur, but I do not know if it works.

**Comment by danarteaga** [comment]
I am using the fork here: https://github.com/notAperson535/OneClick-macOS-Simple-KVM. While the UI is nice, I am still unable to update to Monterey from Catalina. After restarting, it is still using Catalina and prompts me to install the Monterey upgrade once again. Is there something that I am missing?

**Comment by notAperson535** [comment]
@danarteaga do you have any info that needs to be transferred to the Monterey vm? Or could you just do a fresh install. If you can just do a fresh install, then that is what I would do.

**Comment by notAperson535** [comment]
@danarteaga Before you fresh install, can you send me a screenshot of opencore after you try to upgrade?

**Comment by danarteaga** [comment]
@notAperson535 when I checked this morning, the Macintosh SSD disk was no longer available on the bootloader screen (perhaps the data got corrupted when I exited from terminal), so I did a fresh install of Monterey on WSL per your (well written) tutorial.  Everything went as expected. However, after the install, the Macintosh SSD disk does not show. It does appear that the Monterey OS (~12GB in size) did install in disk utility. Any thoughts?

![2](https://user-images.githubusercontent.com/9701772/172215618-d448e751-00fc-4000-bb8c-01e8c6526509.png)
![1](https://user-images.githubusercontent.com/9701772/172215623-d6cb8035-1bdf-482f-a6b6-1b663d5a4725.png)

**Comment by notAperson535** [comment]
@danarteaga upgrading to Monterey using WSL always gave me problems. I would stick to big sure as I wrote here

> I just used Big Sur as it works better. All of the main features in Monterey are either video call features or Safari, which you can get in Big Sur.

**Comment by notAperson535** [comment]
> so I did a fresh install of Monterey on WSL per your (well written) 

Thanks! I tried to make it as easy as possible to understand, because even the old guides were a little confusing

**Comment by danarteaga** [comment]
@notAperson535 unfortunately I'm still having the same issues with Big Sur. I formatted the disc both as APFS and extended (journaled).

**Comment by notAperson535** [comment]
You have to format it as macOS extended journaled (not case ***@***.*** unfortunately I'm still having the same issues with Big Sur. I formatted the disc both as APFS and extended (journaled).

**Comment by notAperson535** [comment]
> @notAperson535 unfortunately I'm still having the same issues with Big Sur. I formatted the disc both as APFS and extended (journaled).

@danarteaga You have to format it as macOS extended journaled (not case sensitive)

**Comment by danarteaga** [comment]
> > @notAperson535 unfortunately I'm still having the same issues with Big Sur. I formatted the disc both as APFS and extended (journaled).
> 
> 
> 
> @danarteaga You have to format it as macOS extended journaled (not case sensitive)

@notAperson535 sorry if I wasn't clear - I formatted it as macOS extended journaled (not case sensitive) and I had the same issue of not being able to select my boot disc on restart. I then tried it with APFS (after another fresh install) and had the same issue once again.

**Comment by notAperson535** [comment]
@danarteaga hmm, try completely deleting the `OneClick-macOS-Simple-KVM` folder and re-download it from github. Also, if you are using a custom iso/img file, try using the one downloaded by the downloader. Also, make sure to do this:
![image](https://user-images.githubusercontent.com/95918679/172366581-416a35d8-5c4c-4426-8ebc-1778d8e4927f.png)

**Comment by notAperson535** [comment]
We should probably move this conversation to [my repo](https://github.com/notAperson535/OneClick-macOS-Simple-KVM)

**Comment by danarteaga** [comment]
@notAperson535 yes, good point.. I actually got it working...I think I wasn't letting the installation complete all the way with all the required restarts? Or it was something else idiosyncratic that I can't fully explain.

**Comment by notAperson535** [comment]
@danarteaga I know what happened. When it restarts, you have to select the macOS installer to finish the installation.

**Comment by danarteaga** [comment]
@notAperson535 yes..that is correct. Also, to bring it back to the original thread conversation, I did end up installing Monterey with your fork as I am requiring Xcode 13 and wanted to stay as up-to-date as possible. It works (Windows 11). Transitions between opening/closing windows are very slow at times, so I adjusted that in system settings. RAM and CPU usage are not high. Chrome hardly works (extremely slow) despite only ~30% RAM usage. Overall, a little laggy with some UI interactions. No other major issues for me.

**Comment by notAperson535** [comment]
@danarteaga see [this](https://oneclick-macos-simple-kvm.notaperson535.is-a.dev/docs/guide-performance) and [this](https://clay-atlas.com/us/blog/2021/08/31/windows-en-wsl-2-memory/)

**Comment by mazen930** [comment]
@notAperson535  i think i am doing something wrong that prevent me from update , but i update then it restart and nothing happens  
This is the last thing it does after i restart then nothing happens , it just open the place where i boot the Catalina version form it
![Screenshot from 2022-06-15 16-26-37](https://user-images.githubusercontent.com/33043493/173854137-7d37d645-f65a-4325-87da-92318b4365a4.png)
